### DISCUSSION TITLE
Make translations easier for contributors

### DISCUSSION THREAD

**Discussion by artandor** [artandor/multistream-tools#24] [Ideas]
It feels like having to create a PR to add translations is a bit discouraging for people that might not know how to do it.

I believe we should either use a cloud based service like [Crowdin](https://crowdin.com/) or create a documentation on how to contribute to translations.

**Comment by comradekingu** [comment]
@artandor In the interest of making things easier for translators, use Weblate instead of Crowdin.

**Reply by vasilvestre** [reply]
How is it easier please ? You already used it or saw it on some popular open project ?
Thank you for the input !

**Reply by comradekingu** [reply]
You can see more languages at once, and navigate between strings.
Moreover, it is copylefted libre software, where Crowdin is a proprietary as-a-service-only platform.
I have used both.
Give https://hosted.weblate.org/hosting/ a go.

**Reply by vasilvestre** [reply]
> You can see more languages at once, and navigate between strings. Moreover, it is copylefted libre software, where Crowdin is a proprietary as-a-service-only platform. I have used both. Give https://hosted.weblate.org/hosting/ a go.

We love open source here, that's a winning point

**Reply by vasilvestre** [reply]
We're actually waiting for approval but as far as I can see, it's perfect so far. Thank you for the proposal :)
I'm closing this when we get approved and I will create an announcement in the Discussions tab.

**Reply by comradekingu** [reply]
Go GPLv3-or-later instead of GPLv3-only? https://hosted.weblate.org/access/multistream-tools/glossary/
Also, don't make the main component a glossary, and allow creation of new languages.

**Reply by vasilvestre** [reply]
> Go GPLv3-or-later instead of GPLv3-only? Also, don't make the main component a glossary, and allow creation of new languages.

I used GPLv3 as it's our license used.
More languages require extra development, we can do this but we may miss new languages so .. I don't know about what to do

**Reply by comradekingu** [reply]
GPLv3+ grants the possibility to go GPLv3- later (or A/GPLv3/4), whereas GPLv3- is locked. Both are GPLv3.
In the interest of making things easier for translators, clicking the button to create the file automatically is better.
More languages clearly better I should think. What is translation if not an option to serve content in the languages users want?

Edit: Also https://github.com/artandor/multistream-tools/pull/45
Nobody wants to read that authoritarian drivel. Bad PR deluxe.

**Reply by vasilvestre** [reply]
Fine let's go GPLv3+

I've enabled it, people may open issue in order to handle new language.

Are you talking about the code of conduct ?

**Reply by comradekingu** [reply]
You have to change it in https://hosted.weblate.org/access/multistream-tools/glossary/
Yes. the CoC is unfriendly hardship wrapped in rosy language. It is a burden to entry, and not a help in building community-community.
